Dragonflight/DawnOfTheInfinite/BlightOfGalakrond.lua

```diff
@@ -14,6 +14,22 @@ mod:SetEncounterID(2668)
 mod:SetRespawnTime(30)
 mod:SetStage(1)
 
+--------------------------------------------------------------------------------
+-- Locals
+--
+
+local corrosiveInfusionCount = 1
+local incineratingBlightbreathCount = 1
+local necrofrostCount = 1
+
+--------------------------------------------------------------------------------
+-- Timers
+--
+
+local timers = {
+	[408141] = {22.3, 16.8, 17.4, 29.2, 16.6, 18.4, 24.8, 16.8, 17.2}, -- Incinerating Blightbreath
```

Comment:
I don't think this one is based on a table, it seems to me it's a repeating pattern (after the 1st one) of [17.0, 17.0, 25.5]. However the catch is that the timed event is the UNIT_SPELLCAST_SUCCEEDED of 413596 (which is at roughly the same time as the voice line "My flame returns!", and then the boss runs to the target player at a fixed speed to actually begin the cast that LW alerts on. So the variations in timer are due to how long it takes to run to the targeted player in both the previous and current cast.

Reply:
none of them seem too great
![](https://i.gyazo.com/6961c77f9f343e4d2ab92809a710b640.png)

feel like they will never be super accurate

Reply:
Probably the most accurate way would be to listen to the UNT event as well as the SPELL_CAST_START event, and subtract the difference in time between the two from the timer when you start it from the SPELL_CAST_START event. Then the timer would fairly reliably end around the point the boss does the yell to start chasing down their target.

Reply:
I leave this to you 🤣 🤣 🤣